dali/pipeline/data/sample_view.h

```diff
@@ -43,23 +43,24 @@ class SampleView {
   /**
    * @brief Return an un-typed pointer to the underlying storage.
    */
-  void *raw_mutable_data() {
+  template <typename ptr_t_ = ptr_t>
+  std::enable_if_t<std::is_same<ptr_t_, void *>::value, void *> _raw_mutable_data() {
     return data_;
   }
 
   /**
    * @brief Return a const, un-typed pointer to the underlying storage.
    */
-  const void *raw_data() const {
+  const void *_raw_data() const {
```

Comment:
Why introducing the underscores?

Reply:
I added a section about naming in the description.

Reply:
Oh, sure that's clever.